PlayerControls/sources/AdVideoControls.swift

```diff
@@ -164,6 +162,7 @@ public final class AdVideoControls: UIViewController {
 
 extension AdVideoControls {
     @IBAction func highlightButton(_ button: UIButton) {
+        guard button.isEnabled, button.alpha == 1 else { return }
```

Comment:
Do we need to check for an alpha?

Reply:
Yeah because skip button now becomes inactive without `isEnabled` property